cpp/src/IceSSL/PluginI.h

```diff
@@ -18,6 +18,7 @@ class ExtendedConnectionInfo : public ConnectionInfo
 public:
 
     TrustError errorCode;
+    std::string host;
```

Comment:
Why can we add this data member without breaking binary compatibility?

Reply:
`ExtendedConnectionInfo` is a new class in 3.7.6 we add it to hold the `TrustError` as we cannot add it to `IceSSL::ConnectionInfo` without breaking binary compatibility.

Reply:
ok